src/components/HOCs/WithLeaderboard/WithLeaderboard.js

```diff
@@ -100,7 +100,8 @@ const WithLeaderboard = function(WrappedComponent, initialMonthsPast=1, initialO
             this.setState({leaderboard})
 
             const userId = _get(this.props, 'user.id')
-            if (userId && !options.ignoreUser && userType !== USER_TYPE_REVIEWER) {
+            const userScore = _get(this.props, 'user.score')
```

Comment:
This is a hack (we know this) and a full paragraph comment is needed to explain the details about why this hack is needed.